lib/tile_bag.rb

```diff
@@ -0,0 +1,60 @@
+module Scrabble
+  class TileBag
+
+    def initialize
+      @bag = bag
+    end
+
+    def bag
+      array = []
+      9.times { array << "A"}
```

Comment:
RE: your cleanup question

All of these lines could read like this:

```ruby
array += ["A"] * 9
```

which might be a little less clunky. However, you're storing a lot of data here and at some point it's just going to be a little cumbersome.